awx/api/templates/api/auth_token_view.md

```diff
@@ -1,4 +1,7 @@
 {% ifmeth POST %}
+
+## DEPRICATED
```

Comment:
Do we need this file at all any more?  It seems like the old view that uses this is totally gone now.

Reply:
Yes, that view is completely gone now, I wasn't sure if we wanted to keep it and mark it deprecated, or let people read the docs & release notes for that.  I'll take this md out.

Reply:
<nit> also it's "deprecated" </nit>

Reply:
Well, in order for it to be deprecated, it still has to exist :)

It looks like we've just removed it, and there's no backwards compatibility, so I say let's just lose this.